database/sql/V077__orders_token_indexes.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX orders_sell_buy_tokens ON orders (sell_token, buy_token);
```

Comment:
A similar composite index already exists:
```
create index order_quoting_parameters
    on orders (sell_token, buy_token, sell_amount);
```
But after adding a separate `(sell_token, buy_token)`, the execution time improved more than twice.

Reply:
Could you please check how 2 indices (one for sell_token and one for buy_token) behaves?

Reply:
There is almost no difference between separate indexes or completely without them.